tests/build_/test_build_request.py

```diff
@@ -205,7 +205,7 @@ def test_render_simple_request(self, registry_uris, build_image):
         assert build_json["spec"]["source"]["git"]["uri"] == TEST_GIT_URI
         assert build_json["spec"]["source"]["git"]["ref"] == TEST_GIT_REF
 
-        expected_output = "john-foo/component:none-20"
+        expected_output = "john-foo/component:none-"
```

Comment:
`random.randrange` is called in `BuildSpec`, so it's tested there. Updated those tests to mock both this and `utcnow()`.

For this test, I think it's enough to check only `BuildRequest` functionality (that output name is prefixed with `registry_uris[0]`, when there is one).